src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Flow.java

```diff
@@ -3233,6 +3256,11 @@ public void analyzeTree(Env<?> env, JCTree tree, TreeMaker make) {
             try {
                 startPos = tree.pos().getStartPosition();
 
+                if (vars == null)
```

Comment:
Curly braces here?

Reply:
This is a copy of the similar code for `vardecls`. I was thinking if I could unify the codepaths, but does not seem to be so simple, so I created a duplicate. So, it might be better to keep the code the same/similar as it was before, to minimize disruptions.